Library/Formula/dcd.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
```

Comment:
Alright.

` :revision ` is necessary because:

* It gives us extra verification and some security usefulness. It is
essentially a checksum for that tag, verifying the tag matches the
expected hash.
* Tags can be changed. Commits are unique.

On 27/03/2015 11:34, John Colvin wrote:
> In Library/Formula/dcd.rb
> <https://github.com/Homebrew/homebrew/pull/38117#discussion_r27288667>:
> 
>> @@ -0,0 +1,13 @@
>> +require "formula"
>> +
>> +class Dcd < Formula
>> +  homepage "https://github.com/Hackerpilot/DCD"
>> +  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
> 
> there are submodules.
> 
> Why is |:revision| necessary as well? The tag specifies everything
> needed, no?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/38117/files#r27288667>.
>